core/src/main/java/org/mapfish/print/config/ConfigurationFactory.java

```diff
@@ -32,7 +37,19 @@
     @PostConstruct
     public final void init() {
         MapfishPrintConstructor constructor = new MapfishPrintConstructor(this.context);
-        this.yaml = new Yaml(constructor);
+
+        LoaderOptions loaderOptions = new LoaderOptions();
+        String maxAliases = System.getenv("YAML_MAX_ALIASES");
```

Comment:
might be worth prefixing with `MF_PRINT` such as `MF_PRINT_YAML_MAX_ALIASES`

Reply:
Just `PRINT_` as in some other places...

Reply:
ok